package.json

```diff
@@ -3,20 +3,25 @@
   "workspaces": {
     "packages": [
       "packages/*"
+    ],
+    "nohoist": [
+      "**/firebase/**"
```

Comment:
We have different versions of firebase in each package, this solves the problem. 
Firebase 8 and 9 are incompatible - they did a breaking changes ... not sure when I will have time to re-write whole web to use firebase 9 .. 😡